src/modules/image/imageApi.ts

```diff
@@ -37,10 +36,11 @@ export const fetchImage = (id: number | string, language?: string): Promise<Imag
   );
 
 export const updateImage = (
+  id: number,
   imageMetadata: UpdatedImageMetadata,
   formData?: FormData,
 ): Promise<ImageApiType> =>
-  fetchAuthorized(`${baseUrl}/${imageMetadata.id}`, {
+  fetchAuthorized(`${baseUrl}/${id}`, {
```

Comment:
Vet ikke helt hva jeg tenker om å dra ut ID fra body. Vil vi helst at det skal passeres inn på egenhånd, eller vil vi passere inn en fullverdig `UpdatedImageMetadata` og basere oss på den?

Reply:
Grunnen til at id'en var i body in the first place var at vi har typet frontend typen feil. Backend typen inneholder ikke `id` og nå som vi tar i bruk backend typer, klager da typescript på at `id` ikke finnes. Så nå gjør jeg det ordentlig og fjerner `id` fra den typen og sender det på en annen vis inn i funksjonen.